leases/models.py

```diff
@@ -84,7 +84,7 @@ class WinterStorageLease(AbstractLease):
         on_delete=models.PROTECT,
         related_name="leases",
     )
-    application = models.ForeignKey(
+    application = models.OneToOneField(
```

Comment:
I think you also would need to rename `related_field` to maybe `lease`?

Reply:
So renaming it from `leases` to `lease`?

Reply:
yes, to describe the updated logic